main_test.go

```diff
@@ -195,6 +195,17 @@ func TestPublishersEndpoints(t *testing.T) {
 			},
 			expectedContentType: "application/json",
 		},
+		{
+			query: "POST /v1/publishers - INVALID PAYLOAD",
```

Comment:
```suggestion
			description: "POST publishers with invalid payload",
			query: "POST /v1/publishers",
```

This is tricky and works almost by chance (we split at space and use the first two strings for the query), but I'd say we should always provide a `description` when we want add some more information to the test.

It's not ideal and we should probably validate `query` more strictly.